From 05d1186b7af9b4daf4c067526ff390d53ac866ae Mon Sep 17 00:00:00 2001
From: Eli Zaretskii <eliz@gnu.org>
Date: Fri, 11 Nov 2022 10:22:29 +0200
Subject: [PATCH] ; Fix a typo in last change.

---
 lisp/progmodes/eglot.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lisp/progmodes/eglot.el b/lisp/progmodes/eglot.el
index 23f6d8e76d43..7c80c8826ade 100644
--- a/eglot.el
+++ b/eglot.el
@@ -1503,7 +1503,7 @@ If optional MARKER, return a marker instead"
   (let ((truepath (file-truename path)))
     (if (url-type (url-generic-parse-url truepath))
         ;; Path is already a URI, so forward it to the LSP server
-        ;; untouched.  The server shopuld be able to handle it, since
+        ;; untouched.  The server should be able to handle it, since
         ;; it provided this URI to clients in the first place.
         truepath
       (concat "file://"
